website/server/controllers/api-v3/tasks.js

```diff
@@ -1243,4 +1249,36 @@ api.deleteTask = {
   },
 };
 
+/**
+ * @api {post} /api/v3/tasks/age-dailies Ages dailies that are in the uesterDailies field
+ * @apiName AgeDailies
+ * @apiGroup Task
+ *
+ *
+ * @apiExample {json} Example call:
+ * curl -X "POST" https://habitica.com/api/v3/tasks/age-dailies
+ *
+ * @apiSuccess {Object} data An empty object
+ *
+ */
+api.ageDailies = {
+  method: 'POST',
+  url: '/tasks/age-dailies',
+  middlewares: [authWithHeaders()],
+  async handler (req, res) {
+    let user = res.locals.user;
+
+    let taskIds = user.yesterDailies;
+    let dailies = await Tasks.Task.find({_id: taskIds, type: 'daily'}).exec();
+    let daysMissed = 1; // I believe we only handle one day damage now. Maybe multi can be for hard mode?
```

Comment:
Cron has support for multiple days to be implemented in future. Ideally, daysMissed would be passed through to this new code as a variable rather than as a fixed `1`